src/features/wallet/SideBarMenu.tsx

```diff
@@ -107,24 +109,34 @@ export function SideBarMenu({
             Transfer History
           </div>
           <div className="flex grow flex-col px-3.5">
-            <div className="grow flex flex-col w-full">
-              {sortedTransfers?.length > 0 &&
-                sortedTransfers.map((t, i) => (
-                  <TransferSummary
-                    key={i}
-                    transfer={t}
-                    onClick={() => {
-                      setSelectedTransfer(t);
-                      setIsModalOpen(true);
-                    }}
-                  />
-                ))}
-            </div>
-            {sortedTransfers?.length > 0 && (
-              <button onClick={resetTransfers} className="my-5 mx-2 flex flex-row items-center">
-                <Image className="mr-4" src={ResetIcon} width={17} height={17} alt="" />
-                <span className="text-gray-900 text-sm font-normal">Reset transaction history</span>
-              </button>
+            {sanctioned ? (
+              <div className="text-gray-800 text-sm font-normal text-center">
+                <span>Unable to pull transaction history</span>
```

Comment:
This sidebar doesn't pull anything. It just surfaces info from the persisted zustand store. A local history of txs.